bucket/tableplus.json

```
@@ -0,0 +1,31 @@
+{
+    "version": "110",
+    "description": "Relational database explorer",
+    "homepage": "https://tableplus.com",
+    "license": "Proprietary",
+    "url": "https://s3.amazonaws.com/tableplus-win-builds/110/TablePlusSetup.exe",
+    "hash": "be696fb0c6f1f39c51cae409e02a364ebcda0172c21cb42c941430f8a9eedce5",
+    "innosetup": true,
+    "architecture" :{
+        "32bit": {
+            "pre_install": "Get-ChildItem \"$dir\" '*,1*' | Rename-Item -NewName { $_.FullName -Replace ',1' }"
```

- Just in really really edge case, when file could contain `,{1|2}` in general filename, not related to inno
- Powersherll Replace operator on objects are defined with lowercase instead of uppercase C# class method.
- Compatibility should not be issue as parameter binding is not converting FIleInfo into string as normal result of `gci` in powershell

```powershell
$fold = "$PSScriptRoot\testFiles"
New-Item $fold -ItemType Directory -ErrorAction SilentlyContinue

foreach ($in in 1..25) {
    foreach ($a in 1..2) {
        $p = Join-Path $fold "$in,$a.test"
        if (-not (Test-Path $p)) { New-Item $p -ItemType File -Force }
    }
}

powershell.exe -Command { get-childItem "$pwd\testFiles" | Rename-Item -NewName { ($_.BaseName -replace ',1', 'PW5' ) + $_.Extension } }
pwsh.exe -Command { get-childItem "$pwd\testFiles" | Rename-Item -NewName { ($_.BaseName -replace ',2', 'PW6' ) + $_.Extension } }
```

```suggestion
            "pre_install": "Get-ChildItem \"$dir\" '*,1.*' | Rename-Item -NewName { $_.FullName -replace ',1\\.', '.' }"
```